### ISSUE TITLE
elseif vs SlevomatCodingStandard.ControlStructures.EarlyExit

### ISSUE THREAD

**Issue by malarzm** [slevomat/coding-standard#306]
After running automatic fixes this code:

```
if ($lower === 'asc') {
    return 1;
} elseif ($lower === 'desc') {
    return -1;
}
```

is changed to:

```
if ($lower === 'asc') {
    return 1;
}

if ($lower === 'desc') {
    return -1;
}
```

Is there any way to prevent such change? We'd rather stick with `elseif`. Cross referencing https://github.com/doctrine/mongodb-odm/pull/1757

**Comment by kukulich** [comment]
@malarzm You can exclude the code `UselessElseIf` but you should probably discuss it in Doctrine CS.

**Comment by malarzm** [comment]
Thanks a lot @kukulich for quick response!

**Comment by malarzm** [comment]
@kukulich by the way am I missing some resources to check existence of switches like `UselessElseIf`? Brief search showed this flag in code only.

**Comment by kukulich** [comment]
@malarzm It's standard PHPCS configuration, you can exclude types of errors you don't want to be reported, see https://github.com/doctrine/coding-standard/blob/master/lib/Doctrine/ruleset.xml#L15

**Comment by malarzm** [comment]
I was more asking for where should I look for such things before opening issue and bothering you :)

**Comment by kukulich** [comment]
You should always run PHPCS with `-s    Show sniff codes in all reports` so you can see what sniff code reports the error.
